en/developer/api/oauth.md

```diff
@@ -1,7 +1,7 @@
 # Creating a new API client
 
 In your wallabag account, you can create a new API client at this URL
-http://localhost:8000/developer/client/create.
+https://app.wallabag.it/developer/client/create.
```

Comment:
Just a suggestion: maybe we should provide only the path `/developer/client/create` here?